src/core/mdn-annotation.js

```diff
@@ -0,0 +1,34 @@
+import { fetchAndCache } from "./utils";
+
+export const name = "core/mdn-annoatation";
+
+const SPEC_MAP_URL =
+  "https://raw.githubusercontent.com/w3c/mdn-spec-links/master/SPECMAP.json";
+const JSON_BASE = "https://w3c.github.io/mdn-spec-links/";
+
+async function fetchAndCacheJson(url, maxAge) {
+  if (!url) return {};
+  const request = new Request(url);
+  const response = await fetchAndCache(request, maxAge);
```

Comment:
Maybe just 
```suggestion
  return fetchAndCache(request, maxAge).then(r => r.json());
```

Reply:
Yes, seems like a separate function may not be necessary here.